### ISSUE TITLE
IE11: single letter navigation "l" do not jump to the list

### ISSUE THREAD

**Issue by surfer0627** [nvaccess/nvda#6986]
Steps to reproduce:
1. Open link
http://jirs.judicial.gov.tw/FJUD/FJUDQRY01M_1.aspx
2. Press l to move to the list.
NVDA report: no next list
3. Press r to move to the first radio button.
4. Press upArrow three times to move to the list.

note:
* In Firefox, press l, NVDA report: list clickable.
* In this particular web page, IE user could not press l to move to next list.

**Comment by bhavyashah** [comment]
@surfer0627  Can you still reproduce the issue as described above with the given test case? At my end, Firefox detects only one list, which is the clickable list, as mentioned earlier by the original reporter. However, in Internet Explorer 11, NVDA can jump to two lists, one with three items and another with two. Bearing in mind the currently active Project Webfix, any inputs? @jcsteh @feerrenrut @leonardder @ehollig

**Comment by feerrenrut** [comment]
@bhavyashah Do you consider this issue fixed for IE in that case? I think this issue needs further clarification before we can consider trying to fix it. Next steps would be to:

- [ ] Produce a minimal test case which triggers the problematic behaviour
- [ ] A summary of actual behaviour with each of the major browsers (Chrome, Firefox, Edge, IE)
- [ ] Clarification on the expected behaviour in each case.

**Comment by bhavyashah** [comment]
If I am following @surfer0627 correctly, then I am unable to reproduce the said behaviour. NVDA does detect a list, two in fact, on the given page in Internet Explorer 11. However, a discrepency worth pointing out here is that there is only one list with 'l' in Firefox and two in IE 11, something worth a technical investigation. I would lean towards prioritizing a quick check as to why different lists are being rendered in these two web browsers so as to understand the scope and magnitude of this issue, and then triage this ticket as a whole.

**Comment by feerrenrut** [comment]
In that case its better to close this issue and open a new one describing that issue directly, there is too much noise on this issue which distracts from the difference between Firefox and IE. Could you please open a new issue to describe the difference in behaviour that you have noticed?

@surfer0627 if you are able to reproduce this and would like to work with us to try to narrow down the cause, then we are happy to re-open this issue.

**Comment by bhavyashah** [comment]
Alright, will do that (not sure when though).

On 8/1/17, Reef Turner <notifications@github.com> wrote:
> In that case its better to close this issue and open a new one describing
> that issue directly, there is too much noise on this issue which distracts
> from the difference between Firefox and IE. Could you please open a new
> issue to describe the difference in behaviour that you have noticed?
>
> @surfer0627 if you are able to reproduce this and would like to work with us
> to try to narrow down the cause, then we are happy to re-open this issue.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/6986#issuecomment-319259329


-- 
Best Regards
Bhavya Shah

Blogger at Hiking Across Horizons: https://bhavyashah125.wordpress.com/

Contacting Me
E-mail Address: bhavya.shah125@gmail.com
Follow me on Twitter @BhavyaShah125 or www.twitter.com/BhavyaShah125
Mobile Number: +91 7506221750
